app/views/main.scala.html

```diff
@@ -2,7 +2,10 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, minimum-scale=1, maximum-scale=1, user-scalable=yes"
```

Comment:
yes, you are right. I didn't find a way to allow user scaling without preventing the browser from scaling the page when rotating the screen. I'm changing the user-scalable option to no to avoid confusion.